buildSrc/src/main/kotlin/Properties.kt

```diff
@@ -7,6 +7,21 @@ fun Project.getProperty(name: String): String? {
 	// sample.var --> SAMPLE_VAR
 	val environmentName = name.toUpperCase().replace(".", "_")
 	val value = findProperty(name)?.toString() ?: System.getenv(environmentName) ?: null
-	logger.info("getProperty($name): $environmentName - found=${!value.isNullOrBlank()}")
+	logger.debug("getProperty($name): $environmentName - found=${!value.isNullOrBlank()}")
 	return value
 }
+
+/**
+ * Helper function to create the SDK version using the `jellyfin.version` and `jellyfin.branch` properties.
+ * Uses [getProperty] to retrieve the properties. Defaults to `latest-SNAPSHOT` if both properties are not set.
+ */
+fun Project.createVersion(): String {
+	val jellyfinVersion = getProperty("jellyfin.version")?.removePrefix("v")
+	val jellyfinBranch = getProperty("jellyfin.branch")?.let { "$it-SNAPSHOT" }
+
+	return when {
+		jellyfinVersion != null -> jellyfinVersion
+		jellyfinBranch != null -> jellyfinBranch
+		else -> "latest-SNAPSHOT"
+	}
```

Comment:
```suggestion
	return jellyfinVersion ?: jellyfinBranch ?: "latest-SNAPSHOT"
```

Reply:
oh yeah like this makes sense actually, My first approach used the elvis operators with the `getProperty` calls